vueui/src/components/view/records/general.vue

```diff
@@ -10,10 +10,10 @@
       <vui-group-item label="Age:"><view-records-field :editable="(editable & 1) > 0" path="active.age"/></vui-group-item>
       <vui-group-item label="Sex:"><view-records-field :editable="(editable & 1) > 0" path="active.sex"/></vui-group-item>
       <vui-group-item label="Rank:"><view-records-field :editable="(editable & 1) > 0" path="active.rank"/></vui-group-item>
-      <vui-group-item label="Phisical Status:">
-        <view-records-field :editable="(editable & 1) > 0" path="active.phisical_status">
+      <vui-group-item label="Physical Status:">
+        <view-records-field :editable="(editable & 1) > 0" path="active.physical_status">
           <select v-model="$root.$data.state.editingvalue">
-            <option v-for="i in choices.phisical_status" :key="i" :value="i">{{ i }}</option>
+            <option v-for="i in choices.phsyical_status" :key="i" :value="i">{{ i }}</option>
```

Comment:
You implemented a different typo instead

Reply:
Yup